dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java

```diff
@@ -122,7 +122,9 @@
 
     public static final int DEFAULT_ALIVE = 60 * 1000;
 
-    public static final int DEFAULT_CONNECTIONS = 0;
+    public static final String DEFAULT_CONNECTIONS = "1";
```

Comment:
int is better.

Reply:
I modified the DEFAULT_CONNECTIONS to see the code, which is not used before. The reason is changed to String type, because it is convenient with ConfigUtils.getProperty, haha